src/main/java/com/raoulvdberge/refinedstorage/tile/TileController.java

```diff
@@ -105,18 +108,17 @@ public Integer getValue(TileController tile) {
 
             for (INetworkNode node : tile.nodeGraph.all()) {
                 if (node.canUpdate()) {
-                    IBlockState state = tile.getWorld().getBlockState(node.getPosition());
+                    ItemStack itemStack = node.getItemStack();
+                    if (itemStack == null) {
```

Comment:
Yes, could be null. `getItemStack()` is `@Nullable` in the interface. I think it would be more convenient for integration in other mods, and that was also a requirement in my 1.10 branch, where most implementations may return null. Only the default capability may return null in this pull request.